src/corehaptics.cs

```diff
@@ -357,6 +362,10 @@ partial interface CHHapticPatternDefinitionKeys {
 		[TV (15,0), NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
 		[Field ("CHHapticPatternKeyEventWaveformUseVolumeEnvelope")]
 		NSString EventWaveformUseVolumeEnvelopeKey { get; }
+
+		[TV (16,0), NoWatch, MacCatalyst (16,0), Mac (13,0), iOS (16,0)]
+		[Field ("CHHapticPatternKeyEventWaveformLoopEnabled")]
+		NSString EventWaveformLoopEnabled { get; }
```

Comment:
Suffix of `Key` to match the other members

```suggestion
		NSString EventWaveformLoopEnabledKey { get; }
```